plenum/test/node_request/message_request/test_valid_message_request.py

```diff
@@ -101,6 +101,9 @@ def test_node_reject_invalid_req_resp_params(looper, nodes):
                                           f.VIEW_NO.nm: 1,
                                           f.PP_SEQ_NO.nm: 10}),
         patched_MessageReq()(LEDGER_STATUS, {f.LEDGER_ID.nm: 100}),
+        patched_MessageReq()(PROPAGATE, {f.IDENTIFIER.nm: 'aa', f.REQ_ID.nm: 'fr'}),
```

Comment:
1) Please split the test into multiple ones for each of the supported message types. It should be clear from the tests what it tests, and what are the requirements (parameters in this case)
2) Please add tests for validation of Message Reply